lib/serverless.js

```diff
@@ -45,12 +70,11 @@ function copyRecursive (
   return pify(cpr)(source, target, CPR_OPTIONS)
 }
 
-function copyRoutes (
+function getRoutes (
   target /* : string */,
   stage /* : string */
-) /* : Promise<void> */ {
+) /* : Promise<Array<ServerlessRouteConfiguration>> */ {
```

Comment:
`ServerlessRouteConfiguration[]` is an alternative to `Array<ServerlessRouteConfiguration>`. No need to change this at all, but this can be useful if you run into a situation where you feel you have too many `<>`.